esignet-insurance.properties

```diff
@@ -469,8 +469,36 @@ mosip.esignet.vci.key-values={\
                                   'order' : {'fullName','policyName','policyExpiresOn','policyIssuedOn','policyNumber','mobile','dob','gender','benefits','email'},\
                                   'background_color': '#FDFAF9',\
                                     'text_color': '#7C4616'}}\
-                 }}\
-        }\
+                 },\
+                  "LifeInsuranceCredential":{\
+                      'format': 'ldp_vc',\
+                      'scope' : 'life_insurance_vc_ldp',\
+                      'cryptographic_binding_methods_supported': {'did:jwk'},\
+                      'cryptographic_suites_supported': {'Ed25519Signature2020'},\
+                      'proof_types_supported': {'jwt'},\
+                      'credential_definition': {\
+                          'type': {'VerifiableCredential'},\
+                          'credentialSubject': {\
+                              'fullName': {'display': {{'name': 'Name','locale': 'en'}}}, \
+                              'mobile': {'display': {{'name': 'Phone Number','locale': 'en'}}},\
+                              'dob': {'display': {{'name': 'Date of Birth','locale': 'en'}}},\
+                              'gender': {'display': {{'name': 'Gender','locale': 'en'}}},\
+                              'benefits': {'display': {{'name': 'Benefits','locale': 'en'}}},\
+                              'email': {'display': {{'name': 'Email Id','locale': 'en'}}},\
+                              'policyIssuedOn': {'display': {{'name': 'Policy Issued On','locale': 'en'}}},\
+                              'policyExpiresOn': {'display': {{'name': 'Policy Expires On','locale': 'en'}}},\
+                              'policyName': {'display': {{'name': 'Policy Name','locale': 'en'}}},\
+                              'policyNumber': {'display': {{'name': 'Policy Number','locale': 'en'}}}\
+                       }},\
+                      'display': {{'name': 'Life Insurance Verifiable Credential', \
+                                    'locale': 'en', \
+                                    'logo': {'url': 'https://sunbird.org/images/sunbird-logo-new.png','alt_text': 'a square logo of a Sunbird'},\
+                                    'background_color': '#FDFAF9',\
+                                    'text_color': '#7C4616',\
+                                    'order' : {'fullName','policyName','policyExpiresOn','policyIssuedOn','policyNumber','mobile','dob','gender','benefits','email'}\
```

Comment:
I think the order needs to be outside display

Reply:
changed this to outside of display